pkg/okteto/pipeline.go

```diff
@@ -88,13 +88,13 @@ func GetPipelineByName(ctx context.Context, name, namespace string) (*PipelineRu
 }
 
 // DeletePipeline deletes a pipeline
-func DeletePipeline(ctx context.Context, name, namespace string) (string, error) {
+func DeletePipeline(ctx context.Context, name, namespace string, destroyVolumes bool) (string, error) {
 	log.Infof("delete pipeline: %s/%s", namespace, name)
 	q := fmt.Sprintf(`mutation{
-		destroyGitRepository(name: "%s", space: "%s"){
+		destroyGitRepository(name: "%s", space: "%s", destroyVolumes: %t){
```

Comment:
do we need to care about API compatibility here?

Reply:
Yes, better to be safe